src/shared/components/blocks/catchphrase/Catchphrase.js

```diff
@@ -1,13 +1,93 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import styles from './Catchphrase.module.css';
+import LayoutContainer from '../../../components/layout-container/LayoutContainer';
 
-const Catchphrase = ({ className }) => (
-    <div className={ classNames(styles.catchphrase, className) }>
-        This is the catchphrase!
-    </div>
-);
+import { debounce } from 'lodash';
+
+class Catchphrase extends Component {
+    constructor(props) {
+        super(props);
+
+        this.isGoingDown = undefined;
+        this.maxLeftTransform = -100;
+        this.maxRightTransform = 0;
+        this.translateSteps = 1.2;
+
+        this.state = {
+            translateValue: 0,
+        };
+    }
+
+    componentDidMount() {
+        window.addEventListener('scroll', debounce(this.handleScroll.bind(this), 10));
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('scroll', this.handleScroll.bind(this));
+    }
+
+    render() {
+        const { className } = this.props;
+        const { translateValue } = this.state;
+
+        const applyClassNames = classNames(
+            styles.scrollable,
+        );
+
+        const applyStyles = {
+            transform: `translate3d(${translateValue}%, -50%, 0)`,
+        };
+
+        return (
+            <LayoutContainer
+                contentClassName={ styles.layoutContent }
+                className={ classNames(styles.container, className) } >
+                <div id="catchphraseContainer" className={ styles.blackScreen }>
```

Comment:
this `id` makes more sense to be in `LayoutContainer`

Reply:
This is the container of the catchphrase itself, so it makes sense imo